### PR TITLE
Infantry 4.0: Saare Tactical Division boogaloo

### PR DESCRIPTION
**Pull request by Lamasmaster**
I would first like to thank people that helped me making this as this is my first major PR:
Carl for original Infantry code
Jack Frost for helping me with coding and guiding me
Alani for mapping entire infantry prep room
Inf Warcrime Sim 2020 (Yacobpo157) for original SoP and Regs

The biggest change in this PR is the fact infantry is getting renamed to Tactical Division with branch changing to SAARE
I changed the loadout of combat technician (CT) to Laser carbine and bulldog bullup as LMG and recoiless rifle are very much broken at the moment
Squad leader is now not defacto leader of infantry but pathfinder but in event of PF being unavailable he is second in command
With infantry now being contractors their SoP and Regs changed and as such no longer should CO/XO be able to abuse them to do security work
SoP and Regs can be found here - https://docs.google.com/document/d/1A2XIah5dlu8YX18zeu9zrzIL0S39VBI9EzRnSW2SbJs/edit?usp=sharing
Infantry prep got reduced in size to not cause problems with merchant shuttle which was present before

### CODE REVIEW

**Comment by Alanii** [comment]
If it is tested and works fine, cool.

Do not  expect this PR to be merged or accepted until staff has deliberated and publicly announced this along with updating the regulations and SoP. Currently Stutters is the only one who can edit both Regulations and SoP, so expect them be the final word on whether or not this gets pushed.

This PR may also require a vote if it becomes an issue.

File: code/modules/modular_computers/file_system/manifest.dm [hunk 1]
```diff
@@ -132,7 +132,7 @@
 		"spt" =   filtered_nano_crew_manifest(SSjobs.titles_by_department(SPT)),
 		"sci" =   filtered_nano_crew_manifest(SSjobs.titles_by_department(SCI)),
 		"sec" =   filtered_nano_crew_manifest(SSjobs.titles_by_department(SEC)),
-		"inf" =   filtered_nano_crew_manifest(SSjobs.titles_by_department(INF)),
+		"INF" =   filtered_nano_crew_manifest(SSjobs.titles_by_department(INF)),
```
Comment (Roland410) [inline code/modules/modular_computers/file_system/manifest.dm:135]: Capitalization here is not necessary, and if we go with Tactical Division should be TAC/TDI or something.

File: maps/torch/job/outfits/hestia_outfits.dm [hunk 2]
```diff
@@ -1,19 +1,19 @@
 /decl/hierarchy/outfit/job/torch/crew/infantry
 	name = OUTFIT_JOB_NAME("Infantry Marine")
-	uniform = /obj/item/clothing/under/solgov/utility/army/exploration
+	uniform = /obj/item/clothing/under/solgov/utility
```
Comment (Roland410) [inline maps/torch/job/outfits/hestia_outfits.dm:3]: SAARE have their own uniform. Applies for the rest.

File: maps/torch/torch_areas.dm [hunk 3]
```diff
@@ -43,6 +43,10 @@
 /area/maintenance/substation/fifthdeck
 	name = "Fifth Deck Substation"
 
+/area/infantry/infprep
+	name = "\improper Tactical Division Preparation Room "
```
Comment (Roland410) [inline maps/torch/torch_areas.dm:47]: Whitespace at the end of the area designation
```suggestion
	name = "\improper Tactical Division Preparation Room"
```

**Comment by Jackofashepard** [comment]
The SAARE uniform looks really bad, i really dont mind contractor utility being the default

File: code/modules/modular_computers/file_system/manifest.dm (hunk 1 above)
Comment (Lamasmaster) [reply code/modules/modular_computers/file_system/manifest.dm:135]: yeah I didnt mean to capitalize that I dont know why it changed

**Comment by Roland410** [comment]
Going to slap DNM until staff are done with the regulations and everything else needed to testmerge/merge this as well. For the WIP tag, let me know when you're fully finished with it and it's ready for a full review.

**Comment by Roland410** [comment]
> The SAARE uniform looks really bad, i really dont mind contractor utility being the default
Feel free to improve it. If they have their specific uniform coded in and working, I personally see no reason not to use it.

**Comment by Roland410** [comment]
It's not that bad. I used to play a SAARE character.

File: maps/torch/torch_areas.dm (hunk 3 above)
Comment (Lamasmaster) [reply maps/torch/torch_areas.dm:47]: Fixed

File: maps/torch/job/outfits/hestia_outfits.dm (hunk 2 above)
Comment (Lamasmaster) [reply maps/torch/job/outfits/hestia_outfits.dm:3]: Changed to SAARE uniform

**Comment by Sparrowbird611** [comment]
So I'm uh, not entirely sure what this is supposed to be. Is this just re adding infantry but renamed and as SAARE contractors?

**Comment by Sparrowbird611** [comment]
Nevermind, I understand now.

I won't beat a dead horse and give all my reasoning against adding infantry, save one. Why do we need a whole new Iraq larper department when we can just merge them with exploration. All the arguments I've heard against that, such as "exploration is the science department" turned out to be false, since science is the actual science department.

All in all, I'm against this. I'd rather work on exploration and give them a few guns/marine and contractor branches, instead of this.

**Comment by BurgerLUA** [comment]
The question on everyone's mind right now is what this PR does to solve the problems that Infantry had.

**Comment by Jackofashepard** [comment]
From what i understand it comes with a entire SOP detailing how the new inf can't valid hunt normal tators/command cant order them to. Boarders are still fair game tho (mercs, ascent, etc)

**Comment by Sparrowbird611** [comment]
If SOP was the only issue with infantry, I don't believe we would even be having this PR in the first place, since they likely would not have been removed. There's far too many issues with having what boils down to either heavier armed exploration or reverse security sitting around. At best, SOP is another thing for admins to look into when an infantry goon screws up and breaks it, but I hardly consider adding more work for our already small admin team a win, and it fails to solve the other numerous issues that plague infantry.

**Pull request closed without merging**